src/processors/utility/FreqBandSplitter.cpp

```diff
@@ -1,7 +1,7 @@
 #include "FreqBandSplitter.h"
 #include "../ParameterHelpers.h"
 
-FreqBandSplitter::FreqBandSplitter (UndoManager* um) : BaseProcessor ("Frequency Splitter", createParameterLayout(), um, 1, numOuts)
+FreqBandSplitter::FreqBandSplitter (UndoManager* um) : BaseProcessor ("Frequency Splitter", createParameterLayout(), um)
```

Comment:
```suggestion
FreqBandSplitter::FreqBandSplitter (UndoManager* um) : BaseProcessor ("Frequency Splitter", createParameterLayout(), um, BasicInputPort, OutputPort)
```

`FreqBandSplitter` already has its own `OutputPort` enum.